docs/source-2.0/aws/rules-engine/built-ins.rst

```diff
@@ -68,6 +68,25 @@ Description
 Type
     ``boolean``
 
+.. _rules-engine-aws-built-ins-account-id:
+
+``AWS::Auth::AccountId`` built-in
+=========================
```

Comment:
These header formatting pieces on both of these need to match the length of the header they're for.

Reply:
Thanks - forgot to run `make html` - fixed (and fixed the duplicate account-id label).